CHANGELOG.md

```diff
@@ -1,13 +1,53 @@
 # TigerBeetle Changelog
 
-## 2023-11-6
+## 2023-11-13
+
+### Safety And Performance
+
+- [#1264](https://github.com/tigerbeetle/tigerbeetle/pull/1264)
+
+  Implement last-mile release artifact verification in CI.
+
+- [#1268](https://github.com/tigerbeetle/tigerbeetle/pull/1268)
+
+  Bump the simulator's safety phase max-ticks to avoid false positives from the liveness check.
+
+- [#1270](https://github.com/tigerbeetle/tigerbeetle/pull/1270)
+
+  Fix a crash caused by a race between a commit and a repair acquiring a client-reply `Write`.
+
+- [#1278](https://github.com/tigerbeetle/tigerbeetle/pull/1278)
+
+  Fix a crash caused by a race between state (table) sync and a move-table compaction.
```

Comment:
Let's work a release song in here! I suggest https://www.youtube.com/watch?v=pq-G3EWO9XM&t=2s. Maybe something like:

> Both bugs didn't stand in the [Line of Fire](https://youtu.be/pq-G3EWO9XM?si=09lLoC1QPdsXWeIV) of our deterministic simulator.